RustFxA/FxAWebView.swift

```diff
@@ -189,6 +197,8 @@ extension FxAWebView: WKScriptMessageHandler {
 
         let auth = FxaAuthData(code: code, state: state, actionQueryParam: "signin")
         RustFirefoxAccounts.shared.accountManager.finishAuthentication(authData: auth) { _ in
+            self.profile.syncManager.onAddedAccount()
```

Comment:
@eoger I added this to do a sync ASAP. (the old code was calling this too)

Reply:
forgot to mention, that func does `self.syncEverything(why: .didLogin)`

Reply:
Awesome, then we can delete the `for` loop above which might create nasty side-effects

Reply:
IIUC, the for-loop sets the UI prefs and is needed, the sync does not set the UI prefs in this case.

Reply:
Based on slack convo, removed for-loop, code works ok